src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -414,6 +414,7 @@ Next, we'll build a package manifest as defined by our manifest:
 
 ```sh
 ${SDK_PATH}/tools/${ARCH}/pm \
+    -api-level 8 \
```

Comment:
```suggestion
    -api-level $(${SDK_PATH}/tools/${ARCH}/ffx version -v | grep "api-level" | head -1 |  awk -F ' ' '{print $2}') \
```

On second pass, this is a programmatic way we can fetch that information. Not the cleanest, so I'm open to suggestions!

Reply:
Yeah, this makes more sense!